Microsoft.Identity.Web/StartupHelpers.cs

```diff
@@ -37,6 +37,18 @@ namespace Microsoft.Identity.Web
 {
     public static class StartupHelpers
     {
+        /// <summary>
+        /// Add authentication with Microsoft identity platform.
+        /// This expects the configuration files will have a section named "AzureAD"
+        /// </summary>
+        /// <param name="services">Service collection to which to add this authentication scheme</param>
+        /// <param name="configuration">The Configuration object</param>
+        /// <returns></returns>
+        public static IServiceCollection AddMicrosoftIdentityPlatformAuthentication(this IServiceCollection services, IConfiguration configuration)
```

Comment:
>AddMicrosoftIdentityPlatformAuthentication [](start = 41, length = 42)

new name suggestion :)

Reply:
We'd want to rather propose a PR for all the samples with the name change?
WIll address this part of building the NuGet package